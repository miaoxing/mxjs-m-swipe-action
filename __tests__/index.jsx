import SwipeAction from '..';
import {render} from '@testing-library/react';
import {init} from 'taro-test';

init();

describe('SwipeAction', () => {
  test('basic', () => {
    const result = render(<SwipeAction/>);
    expect(result.container).toMatchSnapshot();
  });
});
