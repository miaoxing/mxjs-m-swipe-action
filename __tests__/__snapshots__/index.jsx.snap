// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SwipeAction basic 1`] = `
<div>
  <taro-view-core
    style="overflow: hidden;"
  >
    <taro-view-core
      class="mx-swipe-action-content"
      style="transform: translate3d(0px, 0, 0);"
    >
      <taro-view-core />
      <taro-view-core
        class="mx-swipe-action-options __mx-swipe-action-options-1"
      />
    </taro-view-core>
  </taro-view-core>
</div>
`;
